src/services/sessions.js

```diff
@@ -109,6 +117,38 @@ module.exports = class SessionsHelper {
 				})
 			}
 
+			const validMenteeIds = menteeIdsToEnroll.filter((id) => typeof id === 'number')
+			if (menteeIdsToEnroll.length != 0 && validMenteeIds.length != 0) {
+				const menteesDetailsInMentor = await mentorExtensionQueries.getMentorsExtensions(menteeIdsToEnroll)
+				const invalidMentorId =
+					menteesDetailsInMentor.invalidMentors.length === 0 ? [] : menteesDetailsInMentor.invalidMentors
+				const menteesDetailsInMentee = await menteeExtensionQueries.getMenteesExtensions(invalidMentorId)
+				if (
+					(menteesDetailsInMentor.validMentors.length === 0) &
+					(menteesDetailsInMentee.validMentees.length === 0)
+				) {
+					return responses.failureResponse({
+						message: 'INVALID_PERMISSION',
```

Comment:
Is this message correct?

Reply:
@nevil-mathew Yes .

Reply:
Change this message. It's not invalid permissions if the mentees added were not found.
Other places, if used, we can change later.